### DISCUSSION TITLE
RavenDB.Client/RavenDB.Database for .net latest version

### DISCUSSION THREAD

**Discussion by shayja** [ravendb/ravendb#12295] [Q&A]
HI, I work with the  latest version of RavenDB and with the latest .net 5.0 version.
The Raven functionality works fine but the are some NuGet warnings  in the build/publish process about the Raven packages:

warning NU1608: Detected package version outside of dependency constraint: RavenDB.Database 3.5.9 requires RavenDB.Client (= 3.5.9) but version RavenDB.Client 5.1.8 was resolved.
 warning NU1701: Package 'RavenDB.Database 3.5.9' was restored using '.NETFramework,Version=v4.6.1, .NETFramework,Version=v4.6.2, .NETFramework,Version=v4.7, .NETFramework,Version=v4.7.1, .NETFramework,Version=v4.7.2, .NETFramework,Version=v4.8' instead of the project target framework 'net5.0'. This package may not be fully compatible with your project.

**Comment by ppekrol** [comment]
Hi, the latest RavenDB version is 5.1.8. You can download standalone version here: https://ravendb.net/download

The `RavenDB.Database` package concerns versions of RavenDB prior 4.0. If you are interested in embedding RavenDB into your application then `RavenDB.Embedded` might be the package that you want (https://www.nuget.org/packages/RavenDB.Embedded/). We have a documentation article describing how to do so: https://ravendb.net/docs/article-page/5.1/Csharp/server/embedded

If you do not want to run your instance, then consider using our Cloud: https://cloud.ravendb.net/

**Reply by shayja** [reply]
Thanks for the answer.
We use the cloud version of RavenDB (Go to prod soon).
Is the package RavenDB.Embedded recommended for the cloud version of RavenDB?

**Reply by ppekrol** [reply]
No, Embedded hosts the RavenDB server inside your application (as a separate process). Since your server is outside (in our Cloud) then `RavenDB.Client` is all that you need (https://www.nuget.org/packages/RavenDB.Client/).
